File: src/main/java/com/rs/java/game/area/multi/PestControlMulti.java
package com.rs.java.game.area.multi;

import com.rs.java.game.area.Area;

public final class PestControlMulti extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Pest Control Multi";
	}

	public PestControlMulti() {
		super(10536);
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.MULTI;
	}

}

File: src/main/java/com/rs/kotlin/game/world/area/areas/multi/PestControlMulti.kt
package com.rs.kotlin.game.world.area.areas.multi

import com.rs.kotlin.game.world.area.Area

class PestControlMulti : Area(10536) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Pest Control Multi"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.MULTI
    }
}
